1.1.1 doc changes

cs-abhishek-shukla opened the pull request:
- Fixed Typo, Grammar, label names, doc structure, and information flow issues

cs-utkarsh-surana commented on release_notes.md:
```diff
@@ -0,0 +1,6 @@
+## Release Notes
```
Release notes are of the previous version, Also the folder structure of release notes is different

cs-abhishek-shukla committed:
1.1.1 doc changes
- Fixed Typo, Grammar, label names, doc structure, and information flow issues
- Included feedback received
